static/js/snap-details-map.js

```diff
@@ -0,0 +1,104 @@
+/* global d3, topojson */
+
+!(function(window, document, d3, topojson) {
+  function renderMap(el, snapData) {
+    var width = 988; // 990 - 1px borders
```

Comment:
Is this from design advice? It would seem more robust if JavaScript were to calculate this width based on the width of a specific element - what if Vanilla decides pages are now 1200px wide? or 800px?

I get that you want the map to always be the same width and then scroll, but it would still be nice if there was a way for that maxWidth to be relative to something. Can you think of any solution?